app/src/utils/apiHelpers.test.tsx

```diff
@@ -0,0 +1,212 @@
+import {
+  getDivisionData,
+  getFeaturedImage,
+  getFeaturedItemData,
+  getHomePageData,
+  getItemsCountFromUUIDs,
+  getNumDigitizedItems,
+  getRandomFeaturedItem,
+} from "./apiHelpers";
+import { apiResponse } from "./apiResponse";
+import defaultFeaturedItem from "../data/defaultFeaturedItemData";
+import {
+  mockFeaturedItemResponse,
+  mockItemResponse,
+} from "__tests__/__mocks__/data/mockApiResponses";
+
+jest.mock("./apiResponse");
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+describe("getHomePageData", () => {
+  it("creates response containing random number and all 7 lanes", async () => {
+    (apiResponse as jest.Mock).mockResolvedValueOnce(
+      Promise.resolve({
+        nyplAPI: {
+          response: {
+            counts: {
+              count: [
+                {
+                  uuid: { $: "test" },
+                  count_value: { $: "10" },
+                },
+              ],
+            },
+          },
+        },
+      })
+    );
+    const result = await getHomePageData();
+    expect(apiResponse as jest.Mock).toHaveBeenCalled();
+    expect([0, 1]).toContain(result.randomNumber);
+    expect(result.lanesWithNumItems.length).toEqual(7);
+    // Test uuid response doesn't match the collection uuid, so all of these should be 0.
+    expect(
+      result.lanesWithNumItems[0].collections[3].numberOfDigitizedItems
+    ).toEqual("0");
+  });
+});
+
+describe("getFeaturedItemData", () => {
+  it("creates response containing featuredItem and numDigitizedItems", async () => {
+    const result = await getFeaturedItemData();
+
+    // Fallback values:
+    expect(result.numberOfDigitizedItems).toEqual("875,861");
+    expect(result.featuredItem.imageID).toEqual(
+      defaultFeaturedItem.production.featuredItem.imageID
+    );
+  });
+});
+
+describe("getDivisionData", () => {
+  it("forms the correct request from params", async () => {
+    const result = await getDivisionData({
+      slug: "testSlug",
```

Comment:
Add a test where `slug` is _not_ passed. In the function, the `if (slug)` is a branch that should be tested (which you did here) but add a test for when the `if` statement is false.

This can conceptually also be, one test for the main divisions page and one test for the division/slug page.